src/main/java/frc/robot/io/Controls.java

```diff
@@ -7,29 +7,28 @@
 
 package frc.robot.io;
 
-import edu.wpi.first.wpilibj.Joystick;
-import edu.wpi.first.wpilibj.buttons.JoystickButton;
-
```

Comment:
The imports go here, not where you moved them to

Reply:
He has the imports moved down below the 
"add the docs here comment"

Reply:
And according to the official Java coding standard, the imports go directly beneath the package